### ISSUE TITLE
Update plugin

### ISSUE THREAD

**Issue by olegys** [MarshallTechnology/video_stream#2]
Good day!
Are you planning to update this plugin?

**Comment by cmarsh940** [comment]
yes there is a update in the works. I can't give you an exact timeline but there will be a update coming.

**Comment by olegys** [comment]
good news !

**Comment by ghost** [comment]
There is an issue when i first start streaming. The camera froze, i have to switch between front and back camera, start streaming again then it works
By the way how's the update going?

**Comment by lightspect** [comment]
Is there any fall back or Exception when video streaming but the connection is lost? I tried streaming then turn off Internet connection, it threw Java IO Exception before I can catch it on Flutter side and crashing the app.

**Comment by olegys** [comment]
@cmarsh940 
Good day!  Any news regarding the update?  On most android devices, it works with errors due to the old version of the camera plugin.

**Comment by olegys** [comment]
@cmarsh940 Any news ?
